modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -0,0 +1,124 @@
+[id="osd-aws-privatelink-firewall-prerequisites"]
+= AWS PrivateLink firewall prerequisites
+
+If you are using a firewall and want to create your cluster using AWS PrivateLink, you must configure your firewall to grant access to the sites that {product-title} requires.
+
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/installing/installation-configuration#configuring-firewall_configuring-firewall[Configuring your firewall for OpenShift Container Platform] for information about granting access to sites that OCP requires.
```

Comment:
@kmccarron-rh Can we _include_ the OCP module, instead of just pointing to it? It's definitely _very_ easy to overlook this!

Reply:
I think we should wait until after GA to make this change. Do you want to open a Jira card to track?

Reply:
https://issues.redhat.com/browse/OSDOCS-2244